callouts_cfg_test_sept_animation.yml

```diff
@@ -0,0 +1,336 @@
+-
+  event_dates:
```

Comment:
I think you'll want to delete this file so it doesn't end up in the `main` branch